src/planscape/existing_projects/views.py

```diff
@@ -1,16 +1,38 @@
-import requests, urllib
+import requests
+import urllib
 from rest_framework.response import Response
 from rest_framework.views import APIView
 from django.views.generic.base import TemplateView
-import requests, urllib
+import requests
```

Comment:
Nit: remove this line and the next, since lines 1 and 2 are duplicates.

Reply:
Done